kustomize/commands/edit/remove/removeconfigmap.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package remove
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kustomize/api/konfig"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v4/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type removeConfigMapOptions struct {
+	configMapNames []string
```

Comment:
suggest changing the name of `configMapNames` to `configMapNamesToRemove` so that it is extra clear in the code below what these are.